.github/workflows/changeset-pr-snapshot-release.yml

```
@@ -0,0 +1,36 @@
+name: Release PR Snapshot
+
+on: pull_request
+
+jobs:
+  snapshot_release:
+    if: github.repository == 'atlassian/changesets'
```

Do you know if `github.repository` actually means the repo that the pull request is from? I really wouldn't be surprised if it meant the repo that the PR is to.

---

Good catch, for the `pull_request` trigger this should be:

```suggestion
    if: github.event.pull_request.head.repo.full_name == 'atlassian/changesets'
```

This will prevent this workflow PR from potentially running when a pull request targets a fork of the Changesets repo. It's a minor thing but I guess still worth doing if this has already been added, right?